README.md

```diff
@@ -51,6 +51,12 @@ emitter.on('foo', onFoo)   // listen
 emitter.off('foo', onFoo)  // unlisten
 ```
 
+### Typescript
+```ts
+import * as mitt from 'mitt';
+let emitter: mitt.Emitter = new mitt();
```

Comment:
It's actually obvious. If you look at the declaration file, you'll see it exports a namespace. At runtime, mitt module is an object with the "default" prop, which means it has a default export. And indeed, mitt has a default export (the source code written in js/flow), but this isn't defined in the declaration file.